line.go

```diff
@@ -69,7 +70,7 @@ func newTradeSettlement(line *bill.Line) *TradeSettlement {
 			// - O = Outside the tax scope
 			// - L = IGIC (Canary Islands)
 			// - M = IPSI (Ceuta/Melilla)
```

Comment:
This context could probably go into the FindTaxCode function or be removed since we make a decision on what to set the value to in the method. What do you think?

Reply:
You mean to delete these comments, right?

Reply:
yes